### DISCUSSION TITLE
Separators/wrappers for footnotes' labels in Rehype

### DISCUSSION THREAD

**Discussion by jetxr** [remarkjs/remark#1226] [Q&A] [answered]
GitHub uses [square brackets for footnotes' labels](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes). Is this configurable/doable in Rehype?

I tried to running `remark-gfm`'s example for the following input:
```markdown
A note[^1][^2]

[^1]: Test note
[^2]: Test note
```

And it outputs:
```html
<p>A note<sup><a href="#user-content-fn-1" id="user-content-fnref-1" data-footnote-ref aria-describedby="footnote-label">1</a></sup><sup><a href="#user-content-fn-2" id="user-content-fnref-2" data-footnote-ref aria-describedby="footnote-label">2</a></sup></p>
```

![image](https://github.com/remarkjs/remark/assets/13171853/80641238-6b67-4234-8523-1b7f0d97433b)

This looks a bit confusing sometimes (1 2, being confused for 12). How would I configure the labels to be comma separated, or be in brackets (like GitHub's example shows)?

**Comment by wooorm** [comment, marked as answer]
You can inspect GH interface to see that they use CSS.
We have the exact same structure as GitHub.
Meaning you can use the same CSS.
